blog comparing online and in-person TTT

raynamharris opened the pull request:
Here's a post I wrote. I would appreciate comments/suggestions/edits from @ChristinaLK and @gvwilson

raynamharris committed:
blog comparing online and in-person TTT

Here's a post I wrote. I would appreciate comments/suggestions/edits from @ChristinaLK and @gvwilson

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+category: ["Instructor Training"]
+---
+
+I have co-taught three instructor training workshops this year (one online and one in-person with Christina Koch and one online with Greg Wilson. 
```
missing closing parenthesis

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+
+| Feature | In-person | Online |
+| :--- | :---: | :---: |  
+networking | excellent | poor | 
```
Was it poor within each room for the online, or just between the rooms?

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+technical difficulties | medium | high |
+
+## Networking 
+**In-person:** I think the most amazing feature of an in person workshop is the networking. There is something about face to face conversation during activities and over coffee, beer, and  dinner that really solidifies personal relations. I became a certified instructor in January 2015 at UC Davis TTT taught by Greg Wilson, Tracy Teal, Bill Mills, and Aleksandra Pawlik. I consider all four of the ...
```
"in-person" (hyphenated)

gvwilson replied:
and "face-to-face" - but maybe that's a Canadian thing

gvwilson replied:
I don't think most people will know what TTT is.

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+## Networking 
+**In-person:** I think the most amazing feature of an in person workshop is the networking. There is something about face to face conversation during activities and over coffee, beer, and  dinner that really solidifies personal relations. I became a certified instructor in January 2015 at UC Davis TTT taught by Greg Wilson, Tracy Teal, Bill Mills, and Aleksandra Pawlik. I consider all four of the ...
+
+**Online:** The first time I noticed a lack of networking for the online classes was during a debriefing session earlier this week. It wasn't until 45 minutes into the call that I realized that two of the participants were learners in one my my workshops! I'm glad I finally made the connection, but it made me realize who hard it is to recognize people when you only see a tiny image of them on you ...
```
"my my" => "of my"

gvwilson replied:
"who hard" => "how hard"

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+**What does this mean for next time?** I think next time I teach in person, I will use the etherpad chat a lot more, especially for quick yes/no responses.
+
+## Time Commitment
+**In-person:** Even though I really enjoy traveling, saying yes to teaching a workshop in a different cities is a huge time commitment. Instead of just saying yes to teaching from 9-4, I'm saying yes to being in a different city for 24 hours a day. I also have to devote a lot of additional time to planing the travel, traveling, and getting reimbursed for expenses.  
```
"a different cities" => "different city"

gvwilson replied:
"planing" => "planning"

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+## Private Communication
+**In-person:** When co-teaching in person, you can easily communicate privately with your co-instructor during the lesson and during the breaks. You never have to worry about wether you are muted or not or if the learners can hear you, and making decisions about wether or not to change the lesson plan on the fly is pretty easy 
+
+**Online:** When co-teaching online, you have to have yet another application open on your computer for private communication. I like Slack a lot for communicating, but it was a little odd when I was screen sharing and some slack notifications came on the screen (yikes!). Email also works, but then I can get distracted by other emails, so this is not ideal. 
```
I'd lose the "yikes!"

gvwilson commented on _posts/2016/10/2016-10-22-instructrortraining.md:
```diff
@@ -0,0 +1,59 @@
+## Technical Difficulties
+**In-person:** I would rate potential for technical difficulties in the classroom as medium. There's always a chance that the projector system isn't optimal or that the internet connection is poor, but usually an expert in the room can come up with a solution or temporary fix on the fly. 
+
+**Online:** This is a real pain and can eat into your teaching time. I've encountered all sorts of issues including bad bad sound, bad video, poor Etherpad accessibility, inability to screen share, among other things. I don't have an answer for this. 
```
"bad bad sound" => "bad sound"

gvwilson replied:
Inconsistent capitalization: "etherpad" vs "Etherpad"

raynamharris committed:
addressing Greg's comments

raynamharris committed:
add my thought on the projector

This is actually what made this thing about writing this in the first place, so I wanted to add a whole section about what using the projector or not.  @gvwilson thoughts?
